crates/fj-window/src/event_loop_handler.rs

```diff
@@ -99,8 +59,42 @@ impl EventLoopHandler {
             }
         }
 
+        let input_event = input_event(
+            &event,
+            &self.window,
+            &self.held_mouse_button,
+            &mut self.viewer.cursor,
+            self.invert_zoom,
+        );
+        if let Some(input_event) = input_event {
+            self.viewer.handle_input_event(input_event);
+        }
```

Comment:
Why did this move up here? Not saying it's a problem, just wondering why.

Reply:
I moved it to avoid a borrow-checker issue where `event` is partially moved when a processed shape is consumed:
```
ModelEvent::ProcessedShape(shape) => {
    self.viewer.handle_shape_update(shape);
    self.status.update_status("Model processed.");
}
```
I couldn't get the compiler suggestions to work so I just moved the borrow in `input_event(&event, ...)` above the partial move.

Reply:
Got it, makes sense!